ui/src/app/views/dataset/component.tsx

```diff
@@ -187,6 +254,16 @@ export const DataSet = withCurrentWorkspace()(class extends React.Component<
     }
   }
 
+  selectDomainValue(domain: Domain, domainValue: DomainValue): void {
+    const origSelected = this.state.selectedValues;
+    const selectObj = {domain: domain, value: domainValue.value};
+    if (fp.includes(selectObj, origSelected)) {
```

Comment:
`includes` uses `===` for comparisons, which only uses reference identity on objects, so this probably isn't working as intended. Is there some kind of unique id that we can use instead of a compound object? If not, then we'll need to use e.g. `some` instead.

Reply:
I was noticing that. There isn't a unique id, unfortunately, but I can use `some`.